Mathlib/MeasureTheory/Function/LocallyIntegrable.lean

```diff
@@ -415,6 +416,11 @@ theorem ContinuousOn.locallyIntegrableOn [IsLocallyFiniteMeasure μ]
   hf.integrableAt_nhdsWithin hK hx
 #align continuous_on.locally_integrable_on ContinuousOn.locallyIntegrableOn
 
+theorem Continuous.integral_pos_of_integrable_nonneg_nonzero [IsOpenPosMeasure μ] {f : X → ℝ}
+    {x : X} (f_cont : Continuous f) (f_int : Integrable f μ) (f_nonneg : 0 ≤ f) (f_x : f x ≠ 0) :
+    0 < (∫ x, f x ∂μ) := (integral_pos_iff_support_of_nonneg f_nonneg f_int).2
+    (IsOpen.measure_pos μ f_cont.isOpen_support ⟨x, f_x⟩)
```

Comment:
```suggestion
    0 < ∫ x, f x ∂μ :=
  (integral_pos_iff_support_of_nonneg f_nonneg f_int).2
    (IsOpen.measure_pos μ f_cont.isOpen_support ⟨x, f_x⟩)
```
If the proof doesn't fit on one line, it's better to separate it clearly from the statement.